apps/annotation/views/annotation_requests.py

```diff
@@ -28,12 +30,14 @@ def post(self, request):
 Fragment: {}
         '''.format(data['url'], data.get('quote'))
 
-        if not (settings.TEST or settings.DEBUG):
+        try:
             send_mail(
-                sender='annotation_requests',
-                # TODO should work when the domain is verified
+                sender='prosba-o-przypis',
                 to_addr='przypispowszechny@gmail.com',
                 subject=subject,
                 text=text,
             )
+        except MailSendException as e:
+            logger.error('Annotation request could not be sent by e-mail: {}'.format(str(e)))
```

Comment:
How about adding to log some information about annotation request URL stored in `data['url']`?

Reply:
Good idea. I'll add it and merge.